src/runner/runESLint.js

```diff
@@ -30,11 +30,18 @@ const getCachedValues = (config, extraOptions) => {
 const runESLint = ({ testPath, config, extraOptions }) => {
   const start = Date.now();
 
+  // Jest <= 25.x
   if (config.setupTestFrameworkScriptFile) {
     // eslint-disable-next-line import/no-dynamic-require,global-require
     require(config.setupTestFrameworkScriptFile);
   }
 
+  // Jest >= 26.x
```

Comment:
It was added in 24

Reply:
As in peerDeps we have 25 & 26 and both supports `setupFilesAfterEnv`, so remove the comments.